### ISSUE TITLE
Is it possible to change desktop resolution?

### ISSUE THREAD

**Issue by lluxx** [nwjs/nw.js#5196]
Is it possible to change resolution of the users computer in-game?
I saw Screen.screens, and tried to change width and height, but failed. Is it read-only or it is not what I thought it is?

**Comment by ghostoy** [comment]
@lluxx It's not possible to change resolution with NW API currently. We'll investigate feasibilities.

**Comment by lluxx** [comment]
Yes, this is essential if we have a graphic intensive app or game. Changing resolution will improve performance.

**Comment by jportoles** [comment]
@ghostoy any chance of making this happening? Would a cash bounty help? No control over resolutions is a pity for games built with NW.js

**Comment by baconface** [comment]
It is possible with NW.js. Just not baked in. And it will require you to include small third party binaries.

For Windows include [Qres](http://qres.sourceforge.net/) and you can use Node.js to execute a command to it and the resolution will change. `QRes.exe /x:800 /y:600`

Linux you can call `xandr -s 800x600`

For OSX include [cscreen](http://www.pyehouse.com/cscreen/) and execute it using `cscreen -d 32 -x 800 -y 600`

The downfall is if your program exits unnaturally (crashes, computer restarts, etc.) the original resolution will not be returned. Of course you will also have to make your code return the original resolution on natural exits.

**Comment by jportoles** [comment]
Ok, but that's kind of a duct tape solution, isn't it? Having to ship an app with third party binaries isn't ideal, these could randomly fail or be blocked by the end user and the main app would be none the wiser. Native support would still be preferable at the end of day.
